src/functions/products/getProductsList/handler.ts

```diff
@@ -0,0 +1,12 @@
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { products } from 'src/mocks/data';
+
+const getProductsList = async (): Promise<APIGatewayProxyResult> => {
```

Comment:
Async/await is not used in this lambda function. 
Main error scenarios are not  handled here.

Reply:
Could you tell me which main error escenario might come here?

Reply:
You could have went with a simple check on the array and if the array is empty, send back 404 ( pretend it failed getting the resource for some reason ) just for the sake of the task. I am aware that it might be redundant in this case but it is specified in task requirements for some reason.